docs/howto/add_confirm_dialog_in_a_form.md

```diff
@@ -0,0 +1,119 @@
+# How-to: Add a confirm dialog in a form
+
+Asking the user for a confirmation before doing a *dangerous* operation is a
+very common pattern. For that purpose, the Hybrid Platform UI application
+defines an easily reusable markup convention that can be used in various
+contexts.
+
+## Confirm dialog markup convention
+
+This confirm dialog convention relies on [the HTML5 `<dialog>`
+element](https://developer.mozilla.org/fr/docs/Web/HTML/Element/dialog) and a
+`<button>` to open the dialog. To be properly styled, the `<dialog>` element
+should have the class `ez-modal`. By default the `<dialog>` element is hidden to
+the user, it becomes visible when the user clicks on a `<button>` having the
+class `ez-js-open-modal` and a `value` attribute containing a selector matching
+it. As soon as the user clicks on an element with the class `ez-js-close-modal`
+the modal is closed.
+
+## Example
+
+### HTML code
+
+Let's take the following form:
+
+
+```html
+<form action="/whatever/action" method="post">
+    <!-- some markup -->
+
+    <button type="submit" name="DangerousOperation">
+        Dangerous operation
+    </button>
+</form>
+```
+
+To show a confirm dialog before doing the *Dangerous operation*, you have to:
+
+1. Replace the *DangerousOperation* `submit` button by a button with:
```

Comment:
I think it's "with", not "by".

And if you make that "with a button having the following attributes:", you can remove "the ... attribute" from the bullet points below, and avoid repeating it.